### DISCUSSION TITLE
My stats are only reading the "readme" file stats

### DISCUSSION THREAD

**Discussion by pedrovonkarma** [anuraghazra/github-readme-stats#2361] [Q&A] [answered]
### Describe the bug

I've noticed that whenever I commit any public or private project, the stats won't increase (I even tried the "include_all_commits=true", but it didn't solve). Then I tested commiting the readme file (where the stats are located) and the number of commits increased by 1. So I can presume that the code is not reading any other project of my profile, only the readme stats itself.
Is there any solution? I asked a few friends and seems no one had this issue, just me
Here's the link for the readme file: https://github.com/PedroVonKarma/PedroVonKarma

**Comment by rickstaa** [comment, marked as answer]
@PedroVonKarma I checked the results we received from the GraphQL API using [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer), and it looks like the card is correct:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
  }
}
```

```graphql
{"login": "PedroVonKarma"}
```

Please be aware that we use a caching period of 4 hours (see https://github.com/anuraghazra/github-readme-stats#common-options) and that we do currently not include the results of organizations (see #1).
